README.md

```diff
@@ -92,14 +90,14 @@ const receiver = new FiveBellsLedgerPlugin({
   password: 'bobbob'
 })
 
-co(function * () {
-  const stopListening = yield ILP.IPR.listen(receiver, {
+(async function () {
+  const stopListening = await ILP.IPR.listen(receiver, {
     secret: Buffer.from('secret', 'utf8')
-  }, (params) => {
-    console.log('got transfer:', params.transfer)
+  }, ({ transfer, fulfill }) => {
+    console.log('got transfer:', transfer)
 
     console.log('fulfilling.')
-    return params.fulfill()
+    return fulfill()
```

Comment:
I think it might be worth making this an async function so that you can `await fulfill()` and then log a statement afterwards that indicates the money has been received. It would also be good to replace the log line above with something more descriptive of what the recipient is doing